### DISCUSSION TITLE
2F1E7999-B973-4675-BC4C-CC0B0ECE3ADF

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#198] [General]
# AC Herculis

AC Herculis is a binary star system, consisting of a dying post-AGB star and a main sequence star, located about 4,572 light-years away in the constellation Hercules. Both stars are surrounded by a debris disk and a large planet that (if its existence is confirmed) would be the very first polar circumbinary planet ever found

https://celestia.mobi/addon?item=2F1E7999-B973-4675-BC4C-CC0B0ECE3ADF

**Comment by Melneifert** [comment]
I would love to see more moons around exoplanets

**Reply by AU-Microscoppi-b** [reply]
Same lol

**Comment by AU-Microscoppi-b** [comment]
This addon is fine

**Reply by kmjordan122313-lgtm** [reply]
agree with you the addon is fine

**Comment by kmjordan122313-lgtm** [comment]
this looks cool!
